### DISCUSSION TITLE
Compressing glb/glTF files with ktx2 using libktx

### DISCUSSION THREAD

**Discussion by tom-deadline** [KhronosGroup/KTX-Software#427] [Q&A] [answered]
Hello,

I want to compress glb/gltf files with ktx2, can anyone help me with some guides? How can I use libktx for this? Or are there any examples that do this?

Thank you in advance.

**Comment by MarkCallow** [comment, marked as answer]
Our [Artist Guide](https://github.com/KhronosGroup/3D-Formats-Guidelines/blob/main/KTXArtistGuide.md) has this information. Please take a look. See the [KTX Landing Page](https://www.khronos.org/ktx/) on khronos.org for links to more useful information about using KTX.
